src/app/global_styles.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { createGlobalStyle, DefaultTheme, useTheme } from "styled-components";
+import {
+    ThemeProvider as MaterialThemeProvider,
+    createTheme as createMaterialTheme,
+    Theme,
+} from "@mui/material";
+import { lightTheme } from "@/app/themes";
+
+const GlobalStyle = createGlobalStyle`    
+    * {
+        box-sizing: border-box;
+        margin: 0;
+    }
+      
+    html, body {
+        height: 100%;
+        width: 100%;
+        font-family: Helvetica, Arial, sans-serif;
+        overflow: hidden;
+    }
+
+    body {
+        color: ${(props) => props.theme.main.foreground[2]};
```

Comment:
mainly because we use a lot of boxes/containers to contain components like form and stuff
It's just like Google form they use purple background and then white part inside, it's just that we use grey instead of purple here!